tests/Unit/CrudPanel/CrudPanelDeleteTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Backpack\CRUD\Tests\Unit\CrudPanel;
+
+use Illuminate\Support\Facades\DB;
+use Backpack\CRUD\Tests\Unit\Models\Article;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+
+class CrudPanelDeleteTest extends BaseDBCrudPanelTest
+{
+    public function testDelete()
+    {
+        $this->markTestIncomplete('Not correctly implemented');
+
+        $this->crudPanel->setModel(Article::class);
+        $article = Article::find(1);
+
+        $wasDeleted = $this->crudPanel->delete($article->id);
+
+        // TODO: the delete method should not convert the returned result to a string
```

Comment:
But that's what it does currently do - are we testing:

1. Does it actually delete?
2. What it "should" do (but doesn't at the moment)?

PS. I agree the return from `delete` is odd.